tests/test_helpers.py

```diff
@@ -105,6 +105,21 @@
      "Please rerun the helper specifying '-n list-index' to update item number 'list-index':\n"
      "1. Maria as a new contact\n"
      ),
+    #UC1
+    (["helper", "list", "people"],
+     "scopatz\nsbillinge\n"),
+    #UC2
+    (["helper", "list", "--kv_filter", "name", "simon"],
+     "sbillinge\n"),
+    #UC3
+    (["helper", "list", "--kv_filter", "name", "simon", "-r", "name", "position"],
+     "sbillinge, name: Simon J. L. Billinge, position: professor\n"),
+    #UC4?
+    (["helper", "list", "people", "-k"],
```

Comment:
is this general or specific?  If you put this in it is not a general lister.  UC4 is a specific example that needs to be supported by giving a specific set of inputs to your general lister

Reply:
would it be like: if the person gives the collection projecta, then it would give: 
--leader : (description)
--group_members: (description)
etc.. 
and for the people collection it would be:
--name: name of the person
--position: position of the person
?